src/main/kotlin/CheckersModel.kt

```diff
@@ -111,21 +117,25 @@ class CheckersModel(val board : CheckersBoard = CheckersBoard(8)) : BaseModel()
         return false
     }
 
-    fun canEat(squareFromCoords : Pair<Int, Int>): Boolean {
+    private fun canEat(squareFromCoords : Pair<Int, Int>): Boolean {
```

Comment:
I don't think that it's good idea to use private modifier here, because of you can use it in AI methods, but now it's OK.